SampleProject/Assets/Scripts/Title.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
```

Comment:
[SA0102] A syntax error has been discovered in file /var/folders/80/vq2yfwcx0d541ghj3n_6qn_h0000gn/T/style_keeper/8d689f42372a6888604fce36ba4bd6dba8ee5c07/SampleProject/Assets/Scripts/Title.cs on line 1.